pkg/policies/policies.go

```diff
@@ -42,13 +42,15 @@ var CustomDirectoriesPolicies = pathpolicy.NewPathPolicies(map[string]pathpolicy
 
 // CustomFilesPolicies is a set of default policies for custom files
 var CustomFilesPolicies = pathpolicy.NewPathPolicies(map[string]pathpolicy.PathPolicy{
-	"/":           {Deny: true},
-	"/etc":        {},
-	"/root":       {},
-	"/etc/fstab":  {Deny: true},
-	"/etc/shadow": {Deny: true},
-	"/etc/passwd": {Deny: true},
-	"/etc/group":  {Deny: true},
+	"/":              {Deny: true},
+	"/etc":           {},
+	"/root":          {},
+	"/usr/local/bin": {},
+	"/var/local/bin": {},
```

Comment:
I know that I initially agreed with this path, but checking the FHS now, it does not seem that having `bin` under `/var/local` is standardized. Can you please provide more reasoning behind adding this path?

Reply:
Sure, we are working on a new feature for the hosted image builder service - firstboot support. Users will be able to add an executable (a script, ansible or python script, anything with shebang) that will be executed during firstboot.

Now, in order to do that, we need to drop two files - a systemd unit and an executable file. We need to put the executable file somewhere. It cannot be `/etc`, however, because files there have `exec_t` SELinux context and there is a transition from `systemd_unit_t` leading to SELinux denials.

Now, to your point, `/var/local/bin` is a typo. Indeed `/var` is meant for data, this was a mistake. Let me fix this.